src/style/index.scss

```diff
@@ -1,11 +1,7 @@
 // if using CSS library, import here
 @import './variables.scss';
-@import './ball-grid-beat.css';
```

Comment:
Anyone know what these files do? I've left them just in case

Reply:
I think these were for the loader overlay for when the menu items were being fetched

Reply:
I'm not sure, the designs don't show a loader - but I'll keep the files just in case and we can add later if needed